src/api/client.go

```diff
@@ -100,46 +113,38 @@ func setupLogs(client *Client, verbose bool) {
 	// Debug full request and response if verbose = true
 	// Secrets are hidden see ClientLogger
 	if verbose {
-		client.http.SetDebug(true)
-		client.http.SetDebugBodyLimit(2 * 1024)
+		client.resty.SetDebug(true)
+		client.resty.SetDebugBodyLimit(2 * 1024)
 	}
 
-	// Log each retry
-	client.http.AddRetryHook(func(response *resty.Response, err error) {
```

Comment:
Musel som trocha prerobit spracovanie HTTP chyb. Pred zmenou sa vratila `err` iba ked to bola sietova chyba ... a napr. http kod `404` sa vratil ako response (z ktoreho sa dalo zistit, ze to spadlo) .... musel som to prerobit, aby to chybu `err` vracalo vzdy ... bez toho sa mi nedarilo napojit dalsie casti kodu.